### PR TITLE
[Documentation] Add warning about trusted proxies when using stripe behind lb with https

### PR DESCRIPTION
**Pull request by reyostallenberg**
| Q               | A
| --------------- | -----
| Branch?         | master
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | -
| License         | MIT

### CODE REVIEW

**Comment by reyostallenberg** [comment]
See https://github.com/Sylius/Sylius/issues/9035#issuecomment-352664787

File: docs/cookbook/payments/stripe.rst [hunk 1]
```diff
@@ -30,6 +30,10 @@ Go to the ``http://localhost:8000/admin/payment-methods/new/stripe_checkout`` ur
 
     If your are not sure how to do it check how we do it :doc:`for Paypal in this cookbook </cookbook/payments/paypal>`.
 
+.. warning::
+
+    When your project is behind a loadbalancer and uses https you probably need to configure `trusted proxies <http://symfony.com/doc/current/deployment/proxies.html>`_. Otherwise the payment will not succeed and the user will endlesly loopback to the payment page without any notice.
```
Comment (venyii) [inline docs/cookbook/payments/stripe.rst]: `endlessly` (missing 's')

**Comment by pamil** [comment]
Could you rebase it to 1.0? :)

**Comment by pamil** [comment]
Rebased and pushed as #9130, thanks @reyostallenberg! :)

**Pull request closed without merging**